src/test/scala/com/high-performance-spark-examples/GoldiLocks/EvaluationTests.scala

```diff
@@ -0,0 +1,50 @@
+package com.highperformancespark.examples.goldilocks
+
+import org.apache.spark._
+import org.apache.spark.rdd.RDD
+import org.scalatest.{BeforeAndAfterAll, FunSuite}
+
+
+class EvaluationTests extends FunSuite with BeforeAndAfterAll {
```

Comment:
you could also just use SharedSparkContext from com.holdenkarau.spark.testing (but if its to illustrate manual testing no stress).